### DISCUSSION TITLE
Configure Screen

### DISCUSSION THREAD

**Discussion by Sonwon1** [osmandapp/OsmAnd#17807] [General]
Configure Screen is not working for me. I think it is likely a bug? For example, when I add Altitude and / or GPS info to the Right Panel it will move to the Left Panel. I also notice that they will also sometimes appear on the wrong profile in the wrong Panel. Is there something I am doing wrong?

Android 13, Samsung Galaxy Tab Active 3.

**Comment by DmitryAlexei** [comment]
Try updating the app to the latest version (4.5.9 from Google play), the problem may have already been fixed.

**Reply by Sonwon1** [reply]
I forgot to include that in my response. I am on version 4.5.9, released 2023-08-01. So that is not my problem.

**Comment by dmpr0** [comment]
Please contact our support at support@osmand.net and provide logcat file, you can find it in Menu – Help – Help improve OsmAnd – Send logcat log

**Reply by Sonwon1** [reply]
okay

**Reply by Sonwon1** [reply]
I sent a logcat file as requested. The response I received (app crashes) did not match the problem.

**Comment by Sonwon1** [comment]
Am I the only one with this problem?

**Reply by DmitryAlexei** [reply]
Similar issues have been already fixed (see https://github.com/osmandapp/OsmAnd/issues/15074)
If it is your case, you can add a comment with short video and steps to reproduce it.

**Reply by Sonwon1** [reply]
Similar but different, my widgets are actually moving from the right panel to the left panel. I sent a catlog to support and they are asking for another one. So I will do a step by step and add another catlog and send to support.

**Comment by Sonwon1** [comment]
I just sent the logcat file with step by step instructions on how to reproduce. Here is what I sent. It would be nice if someone could give this a try and see if they also have a problem.

Steps to reproduce.
1. start OsmAnd
2. Main Menu
3. Configure Screen
4. right panel (I am on global)
5. add GPS info widget
6. exit widgets (back arrow)
7. exit configure screen (back arrow)
8. change profile to auto
9. change profile back to global

And now the GPS info widget is on the left panel.

**Comment by Sonwon1** [comment]
Okay, I think I have narrowed this problem down a bit more. It works fine on my phone which is running Android 10. The problem occurs on the Android 13, Samsung Galaxy Tab Active 3.

**Comment by Sonwon1** [comment]
Since OsmAnd works correctly on the phone should I wipe it from the tablet and start over?

**Reply by DmitryAlexei** [reply]
Try reinstalling the app, don't forget to previously back up the necessary data (https://osmand.net/docs/user/personal/import-export).
Your issue may be caused by corrupted update of the app. 
If the problem persists, we may need additional data for future investigation (short video, logcat file, profile settings)

**Reply by Sonwon1** [reply]
OsmAnd uninstall and reloaded, same problem. I followed the steps listed above. It appears to me the problem is either tablet related or Android 13.

I have sent the logcat file twice.

I do not know how to send the profile settings.

Please advise.

**Comment by Sonwon1** [comment]
I noticed there is a new version, 4.5.10. This did not fix the problem.

I then noticed the Edit and thought what if I did an Edit and made a simple change and applied the edit. It worked! Now I think I can setup the Panels without OsmAnd rearranging them. I will post here again after I am done if this workaround works or not.

**Comment by Sonwon1** [comment]
The workaround I posted works although I have not road tested, maybe tomorrow if the weather is good.

Another quirked I noticed is the left panel widgets did not show after I set them up until I forced ended OsmAnd and restarted.

**Comment by Sonwon1** [comment]
The workaround works
